dump/src/util/dump/ExternalizationHelper.java

```diff
@@ -1033,6 +1031,61 @@ public void writeTo( ObjectOutput out ) throws IOException {
 
    static class ClassConfig {
 
+      private static Method findGetter( Method[] methods, String name, Class<?> returnType ) throws NoSuchMethodException {
+         for ( Method m : methods ) {
+            if ( !m.getName().equals(name) ) {
+               continue;
+            }
+            if ( !isGetter(m, returnType) ) {
+               continue;
+            }
+            return m;
+         }
+         throw new NoSuchMethodException();
```

Comment:
I think it's best to add the expected name and return type here into the message.